Update dependency: deps/llvm-backend_release

rv-jenkins opened the pull request.

rv-auditor committed:
deps/llvm-backend_release: Set Version 0.1.70

rv-auditor committed:
flake.nix, llvm-backend/src/main/native/llvm-backend: update to version v0.1.70

rv-auditor committed:
flake.lock: update

rv-auditor committed:
deps/llvm-backend_release: Set Version 0.1.71

rv-auditor committed:
flake.nix, llvm-backend/src/main/native/llvm-backend: update to version v0.1.71

rv-auditor committed:
flake.lock: update

rv-auditor committed:
Merge remote-tracking branch 'origin/develop' into _update-deps/runtimeverification/llvm-backend

rv-auditor committed:
deps/llvm-backend_release: Set Version 0.1.72

rv-auditor committed:
flake.nix, llvm-backend/src/main/native/llvm-backend: update to version v0.1.72

rv-auditor committed:
flake.lock: update

rv-auditor committed:
Merge remote-tracking branch 'origin/develop' into _update-deps/runtimeverification/llvm-backend

rv-auditor committed:
deps/llvm-backend_release: Set Version 0.1.73

rv-auditor committed:
flake.nix, llvm-backend/src/main/native/llvm-backend: update to version v0.1.73

rv-auditor committed:
flake.lock: update

Commit:
add missing build dependency

rv-auditor committed:
deps/llvm-backend_release: Set Version 0.1.74

rv-auditor committed:
flake.nix, llvm-backend/src/main/native/llvm-backend: update to version v0.1.74

rv-auditor committed:
flake.lock: update

rv-auditor committed:
deps/llvm-backend_release: Set Version 0.1.75

rv-auditor committed:
flake.nix, llvm-backend/src/main/native/llvm-backend: update to version v0.1.75

rv-auditor committed:
flake.lock: update

Robertorosmaninho commented on install-build-deps:
```diff
@@ -34,6 +34,7 @@ inst_Debian() {
         pkg-config              \
         python3                 \
         python3-dev             \
+        xxd                     \
```
Hey, @dwightguth, could you please add it to this section of [README.md](https://github.com/runtimeverification/k/blob/master/README.md#the-long-version)

Commit:
Update README.md

dwightguth commented on pyk/regression-new/issue-1633/2.test:
```diff
@@ -1 +1 @@
-foo0\uffff\Uffffffff
+foo0\\uffff\\Uffffffff
```
I don't entirely understand why this test was changed. The test was written to imitate the lexical syntax of C, and while it doesn't necessarily break anything to change it, if I don't understand why it was changed, it might be hiding a difference in behavior that might break real semantics.

Robertorosmaninho replied:
The issue here was that it was being interpreted as a Unicode character, but even though it can be parsed as Unicode, it isn't a valid Unicode. After discussing with Scott and Tamas, we'll go with a more limited modification that doesn't require a change to this test but prohibits any Unicode non-latin-1 encoded.

dwightguth replied:
Well it's good that we're not changing this test, so I guess it's moot, but I find your answer confusing because this test never used any unicode characters; that was a literal backslash character. You changed it from a literal backslash character to two literal backslash characters, which was strange enough that I felt the need to comment on it.

Robertorosmaninho committed:
Encoding and decoding llvm pattern's functions
Modifying string tests to only test non-unicode string for bindings.
Decoding `pattern.contents` in `test_string`

Commit:
Merge branch 'develop' into _update-deps/runtimeverification/llvm-backend
